tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionDebugging.cs

```diff
@@ -77,9 +77,22 @@ private static void InitSnapshotsSink()
                 uri => uri);
             var discoveryService = tracer.TracerManager.DiscoveryService;
             var gitMetadataTagsProvider = tracer.TracerManager.GitMetadataTagsProvider;
-            var batchApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider);
+
+            var snapshotBatchUploadApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider, false);
+            var snapshotBatchUploader = BatchUploader.Create(snapshotBatchUploadApi);
+
+            var diagnosticsBatchUploadApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider, true);
+            var diagnosticsBatchUploader = BatchUploader.Create(diagnosticsBatchUploadApi);
+
+            var batchApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider, false);
             var batchUploader = BatchUploader.Create(batchApi);
```

Comment:
I guess you can get rid of it? `batchUploader` used to be passed to `DebuggerSink.Create`, but now it doesn't